example/Fish.md

```diff
@@ -0,0 +1,32 @@
+# Class: Fish
+**Inherits:** Object
+    
+**Defined in:** example_yard.rb
+
+The base class for all fish.
+
+# Public Instance Methods
+## make_sound() [](#method-i-make_sound)
+Make a sound.
+
+**return** [void] 
+**yield** [sound] The sound produced by the fish
+**yieldparam** [String] The actual sound
+## swim(direction, speed) [](#method-i-swim)
+Swim in a specific direction.
+
+Swimming is the most critical feature of fish.
+
+**example** [] swim(:north, 30)
```

Comment:
**issue (blocking):** @example should be formatted differently, not as other tags, and @example text itself should be a code block (with proper highligthing).